client/assets/main.coffee

```diff
@@ -7,6 +7,14 @@ Backbone = require 'backbone'
 window.jQuery = window.$ = $ = require 'jquery'
 window.global = window
 
+React = require 'react'
+DOM = require 'react-dom-factories'
+createClass = require('create-react-class')
+
+# Patch React 16 with deprecated helpers. Can remove once CoffeeScript goes away
+React.DOM = DOM
+React.createClass = createClass
```

Comment:
Oh nice, we were wondering about how we would do this :)

Reply:
❤️ JavaScript